streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBSegmentedBytesStoreTest.java

```diff
@@ -60,23 +70,31 @@
     private final String storeName = "bytes-store";
     private RocksDBSegmentedBytesStore bytesStore;
     private File stateDir;
-    private final SessionKeySchema schema = new SessionKeySchema();
+    private final long windowSizeForTimeWindow = 500L;
+
+    @Parameter
+    public SegmentedBytesStore.KeySchema schema;
+
+    @Parameters(name = "{0}")
+    public static Object[] getKeySchemas() {
+        return new Object[]{new SessionKeySchema(), new WindowKeySchema()};
+    }
 
     @Before
     public void before() {
         schema.init("topic");
         bytesStore = new RocksDBSegmentedBytesStore(storeName,
-                                                    retention,
-                                                    numSegments,
-                                                    schema);
+                retention,
```

Comment:
I'd suggest creating the windows in this function based on the schema type at once, and use them everywhere in the test functions. Looking through the tests, I think three windows are good enough for all of them: `[10, 10]`, `[500, 1000]`, and `[1000, 1500]` as long as we reduce the retention time to 1000L.

Reply:
done